src/GitElephant/Objects/Log.php

```diff
@@ -77,14 +76,8 @@ public static function createFromOutputLines(Repository $repository, array $outp
      * @param int|null    $offset
      * @param bool        $firstParent
      */
-    public function __construct(
-        Repository $repository,
-        $ref = 'HEAD',
-        $path = null,
-        int $limit = 15,
-        int $offset = null,
-        bool $firstParent = false
-    ) {
+    public function __construct(Repository $repository, $ref = 'HEAD', $path = null, int $limit = 15, int $offset = null, bool $firstParent = false)
```

Comment:
Hmm, by recommendations line length shouldn't exceed 80 chars, it's used for improved readability, including here,online code-review in a browser window )